src/lem/slot.rs

```diff
@@ -227,13 +229,20 @@ impl Block {
     }
 }
 
+#[derive(Clone, Debug)]
+pub enum PreimageData<F: LurkField> {
```

Comment:
Can we rename this to `SlotData`?

Reply:
Hmm I think `SlotPreimage` makes more sense

Reply:
The problem I see with "preimage" is that, for bit decomposition, the finite field element looks more like an image than a preimage to me